java/arcs/sdk/Particle.kt

```diff
@@ -11,6 +11,16 @@
 
 package arcs.sdk
 
+typealias Entity = arcs.core.storage.api.Entity
+typealias EntitySpec<T> = arcs.core.storage.api.EntitySpec<T>
+typealias Handle = arcs.core.storage.api.Handle
+typealias ReadableSingleton<T> = arcs.core.storage.api.ReadableSingleton<T>
+typealias WritableSingleton<T> = arcs.core.storage.api.WritableSingleton<T>
+typealias ReadWriteSingleton<T> = arcs.core.storage.api.ReadWriteSingleton<T>
+typealias ReadableCollection<T> = arcs.core.storage.api.ReadableCollection<T>
+typealias WritableCollection<T> = arcs.core.storage.api.WritableCollection<T>
+typealias ReadWriteCollection<T> = arcs.core.storage.api.ReadWriteCollection<T>
+
```

Comment:
These are not particles... i feel like they should be in their own areas.  (E.g. an 'Entity.kt' and a 'Handle.kt', at least).   Also, since these are public-facing APIs now, they really should have some decent KDoc on them.

Reply:
One file per typealias? That seems like overkill, but I agree, the name is wrong.  Perhaps just put them in Api.kt? or Sdk.kt?

Reply:
On 2nd thought, putting the Entity ones in Entity.kt and the Handle ones in Handle.kt is a good trade off.